app/models/line_item.rb

```diff
@@ -25,15 +25,10 @@ class LineItem < FlexCommerceApi::ApiBase
     # @TODO Document other popular methods that we will support
 
     has_one :item
-    belongs_to :container
+    belongs_to :cart, class_name: "::FlexCommerce::Cart"
+
```

Comment:
Will this limit line item fetching to just carts? Should we consider orders as well?

Reply:
Yes it would if we were to use the LineItem gem model to try to fetch an an orders line items e.g `FlexComerce::LineItem.where(cart_id: order.id).find(line_item.id)`. Currently we wouldn't do this we would use the `Order` model and include the line items which is unaffected.